modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/viewmodel/DoneViewModel.kt

```diff
@@ -1,32 +1,31 @@
 package au.com.shiftyjelly.pocketcasts.account.viewmodel
 
-import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import au.com.shiftyjelly.pocketcasts.account.AccountActivity.AccountUpdatedSource
+import au.com.shiftyjelly.pocketcasts.account.R.drawable.ic_email_address_changed
+import au.com.shiftyjelly.pocketcasts.account.R.drawable.ic_password_changed
```

Comment:
Can we use import aliases for resources?
Like: 
`au.com.shiftyjelly.pocketcasts.account.R as AR `
`au.com.shiftyjelly.pocketcasts.localization.R as LR`